packages/widgetbook/pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   flutter:
     sdk: flutter
   freezed_annotation: ^2.0.3
-  go_router: ^3.0.5
+  go_router: ">=^3.0.5"
```

Comment:
Hey Scott, thanks for your contribution, we highly appreciate your efforts 👍🏻

It looks like `go_router: ">=^3.0.5"` is not a valid yaml syntax and `flutter pub get` returns

> Invalid version constraint: Expected version number after ">=" in ">=^3.0.5", got "^3.0.5".

Can you adjust the line to `go_router: ">=3.0.5"` so we can merge your contribution?